test/parallel/test-process-setup-cpu-usage-error.js

```diff
@@ -0,0 +1,16 @@
+// Flags: --expose-internals
+'use strict';
+
+const common = require('../common');
+
+const internalProcess = require('internal/process');
+
+const errMsg = 'setup_cpuUsage failed';
+process.cpuUsage = () => { throw new Error(errMsg); };
+
+internalProcess.setup_cpuUsage();
+
+common.expectsError(() => process.cpuUsage(), {
+  type: Error,
+  message: errMsg
```

Comment:
This should be:

```js
common.expectsError(() => process.cpuUsage(), {
  code: 'ERR_CPU_USAGE',
  type: 'Error',
  message: 'Unable to obtain cpu usage test'
});
```

as per

https://github.com/nodejs/node/blob/90640f7bf355bb2cec3cf90f12ac28741eb68420/lib/internal/process.js#L41-L44

Reply:
Ya oddly this is failing with:

Path: parallel/test-process-setup-cpu-usage-error
assert.js:42
  throw new errors.AssertionError({
  ^

AssertionError [ERR_ASSERTION]: undefined === 'ERR_CPU_USAGE'
    at Object.<anonymous> (/Users/kaila/Developer/Node/test/common/index.js:727:12)
    at Object.<anonymous> (/Users/kaila/Developer/Node/test/common/index.js:533:15)
    at expectedException (assert.js:154:19)
    at innerThrows (assert.js:188:21)
    at Function.throws (assert.js:205:3)
    at Object.expectsError (/Users/kaila/Developer/Node/test/common/index.js:761:12)
    at Object.<anonymous> (/Users/kaila/Developer/Node/test/parallel/test-process-setup-cpu-usage-error.js:13:8)
    at Module._compile (module.js:616:30)
    at Object.Module._extensions..js (module.js:627:10)
    at Module.load (module.js:535:32)
Command: out/Release/node --expose-internals /Users/kaila/Developer/Node/test/parallel/test-process-setup-cpu-usage-error.js
[01:12|%  89|+ 1826|-   1]: Done        ^C                                     
make: *** [test] Error 1

I don't know why this should be the case as I verified that source code matches what you pasted up there. Before I was referencing the backported branch code. Looking into this.

Reply:
Debugging this yields that internal/process.js has the following code:

// Call the native function to get the current values.
    const errmsg = _cpuUsage(cpuValues);
    if (errmsg) {
      throw new Error('unable to obtain CPU usage: ' + errmsg);
    }

I did make clean and then make test -j4 but still this is the case... odd...

Reply:
Sounds like you're on the v8.x branchSounds like you're on the v8.x branch